.github/workflows/eccoBuild.yml

```diff
@@ -0,0 +1,34 @@
+# This workflow uses actions that are not certified by GitHub.
```

Comment:
there is already a build-workflow for ecco, so this file can probably be removed

Reply:
You are right, didn't we delete that already?